src/hotspot/share/oops/fieldStreams.inline.hpp

```diff
@@ -25,31 +25,41 @@
 #ifndef SHARE_OOPS_FIELDSTREAMS_INLINE_HPP
 #define SHARE_OOPS_FIELDSTREAMS_INLINE_HPP
 
+#include "oops/fieldInfo.inline.hpp"
```

Comment:
I don't know if you have to include oops/fieldInfo.inline.hpp but the include line for fieldStreams.hpp should be by itself and then this new include should be below with runtime/javaThread.hpp